### ISSUE TITLE
Do other pretrained checkpoints exist?

### ISSUE THREAD

**Issue by gmlwns2000** [ChenyangQiQi/BSVD#3]
Hi,

## BSVD-32?
> In the following, “BSVD-32” denotes our model whose smallest feature channel is “32”.

However, in the repository, I cannot find the checkpoint of BSVD-32. Can you provide it?
I wanna insert this model into my realtime pipeline, however BSVD-64 seems bit slower than realtime requirement.

## Optimization?
Took about 110ms for a 1x3x4x960x540 image with 3080@FP16; I wanna reduce this by at least about 15ms. I did not apply TensorRT yet.
For my requirement is it good to train a new model with much lower channels? (about 16??)

## How to generate input?
Also I think I implement the wrong model input, and I cannot find good example in the code. Can you suggest any example?

Thank you for your great work!

**Comment by ChenyangQiQi** [comment]
Hi, please check the BSVD-32 at https://hkustconnect-my.sharepoint.com/:f:/g/personal/cqiaa_connect_ust_hk/Em-latu2Zm1MpPoxstOmpCQBzNTkyGVqdUEODK3oxcz6eA?e=loC1pu;

To satisfy your lower running time requirement, the easiest way is to apply TensorRT. I am so sure the performance with lower channels.

You may check the dataloader code about input data
https://github.com/ChenyangQiQi/BSVD/blob/main/Experimental_root/data/video_dali_dataset.py

**Comment by gmlwns2000** [comment]
Hi,

Thank you for your response!

I tried the [tensor RT implementation](https://github.com/gmlwns2000/sharkshark-4k/commits/bsvd-trt), but unfortunately, the official implementation could not be translated into tensor RT because some data-dependent control flows could not be translated in tensorRT.

However, thank you so much for providing 32 channel BSVD checkpoint. 

For model input, what is the noise_map? Is it strength to denoise in each pixel? As I learned in empirically, BSVD returns denoise part of image that large value of noise_map area. Is this correct?

Thanks!
